ITEM.tsv

```diff
@@ -8523,261 +8523,261 @@ ITEM_20151102_008522	Unsent Letters
+ITEM_20151224_008626	Superior Royal Mage Gauntlet
+ITEM_20151224_008627	Superior Royal Mage Boots
+ITEM_20151224_008628	Superior Royal Mage Pants
+ITEM_20151224_008629	Superior Royal Mage Robe
+ITEM_20151224_008630	Superior Bandit Glove
+ITEM_20151224_008631	Superior Bandit Boots
+ITEM_20151224_008632	Superior Bandit Pants
+ITEM_20151224_008633	Superior Bandit Armor
+ITEM_20151224_008634	Superior Royal Guard Gauntlet
+ITEM_20151224_008635	Superior Royal Guard Greaves
+ITEM_20151224_008636	Superior Royal Guard Pants
+ITEM_20151224_008637	Superior Royal Guard Armor
+ITEM_20151224_008638	Belnia Monkey Leather Pants
+ITEM_20151224_008639	Belnia Monkey Leather Armor
+ITEM_20151224_008640	Elgos Glove
+ITEM_20151224_008641	Elgos Boots
+ITEM_20151224_008642	Elgos Pants
+ITEM_20151224_008643	Elgos Robe
+ITEM_20151224_008644	Evre Glove
+ITEM_20151224_008645	Evre Boots
+ITEM_20151224_008646	Evre Pants
+ITEM_20151224_008647	Evre Armor
+ITEM_20151224_008648	Grinas Gauntlet
+ITEM_20151224_008649	Grinas Greaves
+ITEM_20151224_008650	Grinas Leggings
+ITEM_20151224_008651	Grinas Armor
+ITEM_20151224_008652	Lets you to run fast as a wind.
+ITEM_20151224_008653	Many people does not know, but this equipment can be acquired so cheaply due to Goddess Laima planning with her old foresight. 
+ITEM_20151224_008654	Superior Sage Necklace
+ITEM_20151224_008655	Guardian Necklace
+ITEM_20151224_008656	Superior Guardian Necklace
+ITEM_20151224_008657	Nebelet Necklace
+ITEM_20151224_008658	Maven Necklace
+ITEM_20151224_008659	 - Hit by enemy's magic attack will activate effect increasing magic defense by {img red_up_arrow 16 16}6 and decreasing physical defense by {img green_down_arrow 16 16}6 (Maximum stack of 20)
+ITEM_20151224_008660	 - Increase magic amplification by 40% of HP recovery{nl} - Decrease basic HP recovery to 0
+ITEM_20151224_008661	Stone Bracelet
+ITEM_20151224_008662	The stone filled with magic can be used to create bracelets.
+ITEM_20151224_008663	Superior Star Bracelet
+ITEM_20151224_008664	Superior Gem Bracelet
+ITEM_20151224_008665	Kranto Bracelet
+ITEM_20151224_008666	 - Defeating plant enemy will activate plant collection effect{nl}  -  Herb items can be droped when the plant collection effect is stacked 15 times
+ITEM_20151224_008667	Elements Dance
+ITEM_20151224_008668	Created by mages to protect element magics, but fatal flaw couldn't be solved.
+ITEM_20151224_008669	 - Avoiding enemy attack will activate frenzy effect{nl}  - Increases physical attack by 235, movement speed by 10 and reduces physical defense by 88 for 30 seconds when the frenzy effect is stacked 8 times.
+ITEM_20151224_008670	Templar Costume
+ITEM_20151224_008671	Sage Costume
+ITEM_20151224_008672	Warlock Costume
+ITEM_20151224_008673	Kabbalist Costume
+ITEM_20151224_008674	Hair Costume 1
+ITEM_20151224_008675	Hair Costume 2
+ITEM_20151224_008676	Hair Costume 3
+ITEM_20151224_008677	Hairpin precisely finished by master craftsman. Right-click to equip.
```

Comment:
A hairpin that is finished with precision with the help of a master craftsman (or, artisan). Right-click to equip.